wgsl/index.bs

```diff
@@ -6349,244 +6362,340 @@ That's not a full user-defined function declaration.
+    <td>Component-wise fraction and exponent.
+    Component |i| of the result is `frexp(`|e1|`[i], &(*`|e2|`).[i])`.
+    (GLSLstd450Frexp)
+
   <tr algorithm="scalar case, inverseSqrt">
     <td>|T| is f32
     <td class="nowrap">`inverseSqrt(`|e|`:` |T| `) -> ` |T|
-    <td>(GLSLstd450InverseSqrt)
+    <td>Returns the reciprocal of `sqrt(`|e|`)`. (GLSLstd450InverseSqrt)
   <tr algorithm="vector case, inverseSqrt">
     <td>|T| is f32
     <td class="nowrap">`inverseSqrt(`|e|`:` vec|N|<|T|> `) -> ` vec|N|<|T|>
-    <td>(GLSLstd450InverseSqrt)
+    <td>Component-wise reciprocal square root.
+    Component |i| of the result is `inverseSqrt(`|e1|`[i])`.
+    (GLSLstd450InverseSqrt)
+
   <tr algorithm="scalar case, ldexp">
     <td>|T| is f32<br>
         |I| is i32 or u32
     <td class="nowrap">`ldexp(`|e1|`:` |T| `, `|e2|`:` |I| `) -> ` |T|
-    <td>(GLSLstd450Ldexp)
+    <td>Returns |e1| `* 2`<sup>|e2|</sup>. (GLSLstd450Ldexp)
   <tr algorithm="vector case, ldexp">
     <td>|T| is f32<br>
         |I| is i32 or u32
     <td class="nowrap">`ldexp(`|e1|`:` vec|N|<|T|> `, `|e2|`:` vec|N|&lt;|I|&gt;`) -> ` vec|N|<|T|>
-    <td>(GLSLstd450Ldexp)
+    <td>Component-wise floating-point number composition.
+    Component |i| of the result is `ldexp(`|e1|`[i], `|e2|`[i])`.
+    (GLSLstd450Ldexp)
+
   <tr algorithm="scalar case, length">
     <td>|T| is f32
     <td class="nowrap">`length(`|e|`:` |T| `) -> ` |T|
-    <td>(GLSLstd450Length)
+    <td>Returns `abs(`|e|`)`.
+    (GLSLstd450Length)
   <tr algorithm="vector case, length">
     <td>|T| is f32
     <td class="nowrap">`length(`|e|`:` vec|N|<|T|> `) -> ` |T|
-    <td>(GLSLstd450Length)
+    <td>Returns the length |e| (e.g. `sqrt(`|e|`[0]`<sup>`2`</sup> `+` |e|`[1]`<sup>`2`</sup> `+ ...)`).
+    (GLSLstd450Length)
+
   <tr algorithm="scalar case, log">
     <td>|T| is f32
     <td class="nowrap">`log(`|e|`:` |T| `) -> ` |T|
-    <td>(GLSLstd450Log)
+    <td>Returns the natural logaritm of |e|. (GLSLstd450Log)
   <tr algorithm="vector case, log">
     <td>|T| is f32
     <td class="nowrap">`log(`|e|`:` vec|N|<|T|> `) -> ` vec|N|<|T|>
-    <td>(GLSLstd450Log)
+    <td>Component-wise natural logarithm.
+    Component |i| of the result is `log(`|e|`[i])`.
+    (GLSLstd450Log)
+
   <tr algorithm="scalar case, log2">
     <td>|T| is f32
     <td class="nowrap">`log2(`|e|`:` |T| `) -> ` |T|
-    <td>(GLSLstd450Log2)
+    <td>Returns the base-2 logarithm of |e|. (GLSLstd450Log2)
   <tr algorithm="vector case, log2">
     <td>|T| is f32
     <td class="nowrap">`log2(`|e|`:` vec|N|<|T|> `) -> ` vec|N|<|T|>
-    <td>(GLSLstd450Log2)
+    <td>Component-wise base-2 logarithm.
+    Component |i| of the result is `log2(`|e|`[i])`.
+    (GLSLstd450Log2)
+
   <tr algorithm="scalar case, max">
     <td>|T| is f32
     <td class="nowrap">`max(`|e1|`:` |T| `, `|e2|`:` |T| `) -> ` |T|
-    <td>(GLSLstd450NMax)
+    <td>Returns |e2| if |e1| is less than |e2|, and |e1| otherwise. (GLSLstd450NMax)
```

Comment:
Something somewhere should spell out the behaviour on NaNs.

I filed https://github.com/gpuweb/gpuweb/issues/1711 for followup.

Reply:
Agreed it should be spelled out and that it should be done separately.